Source/SA/RFmx WLAN/RFmxWLAN.cs

```diff
@@ -343,14 +344,8 @@ public static void ConfigureSEM(RFmxWlanMX wlanSignal, SEMConfiguration semConfi
             wlanSignal.Sem.Configuration.ConfigureAveraging(selectorString, semConfig.AveragingEnabled, semConfig.AveragingCount, semConfig.AveragingType);
             wlanSignal.Sem.Configuration.ConfigureSpan(selectorString, semConfig.SpanAuto, semConfig.Span_Hz);
 
-            switch (semConfig.MaskType)
-            {
-                case RFmxWlanMXSemMaskType.Standard:
-                    wlanSignal.Sem.Configuration.ConfigureMaskType(selectorString, semConfig.MaskType);
-                    break;
-                default:
-                    throw new System.NotImplementedException("Custom SEM Mask configurations have not been implemented in this Reference Design module.");
-            }
+            // Support for custom masks has not been implemented in this module
+            wlanSignal.Sem.Configuration.ConfigureMaskType(selectorString, RFmxWlanMXSemMaskType.Standard);

```

Comment:
Why have a configurable mask type in the sem config if you hardcode the mask type?

Reply:
I think you're confusing what was previously implemented vs. my changes. Previously, the user had the option to select the mask type in SEMConfig; but, if they selected Custom they would get a run-time error because the developer hadn't implemented it.

I thought it was better to simply not expose the property so that it is clearly something the user will have to implement manually. Hence, at this point the mask type is not a configurable property because it is being set explicitly. [See this line in the diff](https://github.com/NISystemsEngineering/rfic-reference-design-libraries-csharp/pull/68/files#diff-f80807f053c4e349067bc310dc185a92L145)

Reply:
my bad you're right